src/sidebar/services/api.js

```diff
@@ -92,18 +89,13 @@ function get(object, path) {
  * @param links - Object or promise for an object mapping named API routes to
  *                URL templates and methods
  * @param route - The dotted path of the named API route (eg. `annotation.create`)
- * @param [APIMethodOptions] - Configuration for the API method
+ * @param {APIMethodOptions} options - Configuration for the API method
```

Comment:
Shouldn't `options` be optional?

Reply:
> Shouldn't options be optional?

You'd expect so, but the naming here was bad. This is really a set of callbacks invoked at various points during the API request. We _could_ make some of them optional, but there is no benefit because this is an internal helper in the module and in practice we're always going to specify all of them.

Reply:
```suggestion
 * @param {APIMethodCallbacks} callbacks - Callbacks for the API method
```